static/coffee/modals.coffee

```diff
@@ -90,13 +90,19 @@ modals.controller 'NewCaseModalController', ['$scope', '$uibModalInstance', 'sum
 #=====================================================================
 # Assign to partner modal
 #=====================================================================
-modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', ($scope, $uibModalInstance, title, prompt, partners) ->
+modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', 'users', 'UserService', ($scope, $uibModalInstance, title, prompt, partners, users, UserService) ->
```

Comment:
The reason we didn't go for that here is that it is an existing modal, and we didn't want to change too much in case it affects the system in ways we don't anticipate. But if you see this as a way to move these modals over to the new way, then I'm happy to give it a try.